rllib/agents/impala/tests/test_impala.py

```diff
@@ -18,6 +18,25 @@ def setUpClass(cls) -> None:
     def tearDownClass(cls) -> None:
         ray.shutdown()
 
+    def test_impala_lr_schedule(self):
+        config = impala.DEFAULT_CONFIG.copy()
+        config["lr_schedule"] = [
+            [0, 0.0005],
+            [10000, 0.000001],
+        ]
+        local_cfg = config.copy()
+        trainer = impala.ImpalaTrainer(config=local_cfg, env="CartPole-v0")
+
+        def get_lr(result):
+            return result["info"]["learner"]["default_policy"]["cur_lr"]
```

Comment:
This is a bit janky... is there a better to access cur_lr?

Reply:
Fine with me, but yeah: `trainer.get_policy().cur_lr` would be shorter.